### ISSUE TITLE
could  add a new function noise() like imageMagick provided?

### ISSUE THREAD

**Issue by ouonet** [lovell/sharp#725]
could  add a new function noise() like imageMagick provided?
thanks.

**Comment by lovell** [comment]
Hello, libvips provides a [gaussnoise](http://www.vips.ecs.soton.ac.uk/supported/current/doc/html/libvips/libvips-create.html#vips-gaussnoise) function for this, which we'd need to expose.

It would make the most sense, i.e. will be a bit easier, to add this after #470.

**Comment by kimmobrunfeldt** [comment]
This would be helpful in our use case as well.

**Comment by jbveepee** [comment]
This feature would be useful in our use cases as well.
The sharp documentation says it's straightforward if libvips has the function... 
I have no experience with C++ but I'll gladly take a look at how to expose the function if you think it has a chance of succeeding. :-)

**Comment by lovell** [comment]
This was added via https://github.com/lovell/sharp/pull/2527 and included in sharp v0.27.1

https://sharp.pixelplumbing.com/changelog#v0271---27th-january-2021

```js
// Generate RGB Gaussian noise
await sharp({
  create: {
    width: 300,
    height: 200,
    channels: 3,
    noise: {
      type: 'gaussian',
      mean: 128,
      sigma: 30
    }
 }
}).toFile('noise.png');
```
